app/services/email_service.rb

```diff
@@ -74,7 +74,7 @@ def url
   end
 
   def current_responsibility
-    ResponsibilityService.
+    HandoverDateService.
       calculate_pom_responsibility(@offender).to_s.downcase
```

Comment:
Same thing here. I think this should be:

```suggestion
    @offender.pom_responsibility.to_s.downcase
```

Otherwise overrides aren't taken into account.

Reply:
same as above - it's mostly harmless as it's display code, but it's probably wrong

Reply:
Fixed by #1435